quantum/rgb_matrix.c

```diff
@@ -546,3 +552,8 @@ void rgb_matrix_sethsv_noeeprom(uint16_t hue, uint8_t sat, uint8_t val) {
     rgb_matrix_config.hsv.v = val;
     if (rgb_matrix_config.hsv.v > RGB_MATRIX_MAXIMUM_BRIGHTNESS) rgb_matrix_config.hsv.v = RGB_MATRIX_MAXIMUM_BRIGHTNESS;
 }
+
+HSV     rgb_matrix_get_hsv(void) { return rgb_matrix_config.hsv; }
```

Comment:
This does not appear to be in the documentation.

Reply:
Correct.  I wasn't sure about this function, and how useful it would actually be.

So, if the function is useful, I can add an entry for it. 
If we feel that it's not useful, I can remove it.

Reply:
I see.

Reply:
Actually, I'll go ahead and document it.